x-pack/plugins/ml/server/models/results_service/results_service.ts

```diff
@@ -96,7 +96,7 @@ export function resultsServiceProvider(mlClient: MlClient, client?: IScopedClust
     dateFormatTz: string,
     maxRecords: number = ANOMALIES_TABLE_DEFAULT_QUERY_SIZE,
     maxExamples: number = DEFAULT_MAX_EXAMPLES,
-    influencersFilterQuery?: any,
+    filterQuery?: any,
```

Comment:
This `getAnomaliesTableData` function is doing a very specific task to build the data for the anomalies table, as used in the ML Anomaly Explorer and Single Metric Viewer. I'd be very wary about trying to use the `influencersFilterQuery` to do more generic filtering than the tasks it is currently designed for - for example using the query entered in the KQL query bar in the Anomaly Explorer bar.

![image](https://user-images.githubusercontent.com/7405507/155126069-bd4a661c-957a-4d1b-9d3d-13d5372d9761.png)